src/constants/pages.ts

```diff
@@ -0,0 +1,3 @@
+export const HOMEPAGE_NAME = "index.md"
```

Comment:
Minor nit: Should we standardise to call it FILENAME?

Reply:
sure!